src/plugins/saved_objects/public/plugin.ts

```diff
@@ -45,6 +56,10 @@ export interface SavedObjectsStart {
      */
     getListingLimit: () => number;
   };
+  getSavedObjectLabel: typeof getSavedObjectLabel;
+  getAllowedTypes: () => Promise<SavedObjectManagementTypeInfo[]>;
+  getSavedObjectsTagging: () => SavedObjectsTaggingApi | undefined;
+  savedObjectsClient: SavedObjectsClientPublicToCommon;
```

Comment:
Follow-up from previous comment: what is the exact intent / usage here? That kind of direct 'contract re-export' is kinda an anti-pattern of our contract-based plugins communication. Would need to understand if that's strictly a necessity.